addons/edi/models/edi_transfer.py

```diff
@@ -163,13 +164,22 @@ def send_outputs(self, conn):
             self.output_ids += outputs
 
     def do_transfer(self, conn):
-        """Receive input attachments, process documents, send outputs"""
+        """Receive input attachments, send outputs"""
```

Comment:
`send outputs if flag is set` ?

Reply:
hmm there was not if flag is set before, did i remove it?